crates/router/src/core/payments.rs

```diff
@@ -1741,6 +1741,33 @@ where
                 ),
             );
         }
+        TokenizationAction::DecryptPazeToken(payment_processing_details) => {
+            let paze_data = match payment_data.get_payment_method_data() {
+                Some(domain::PaymentMethodData::Wallet(domain::WalletData::Paze(wallet_data))) => {
+                    Some(
+                        decrypt_paze_token(
+                            wallet_data.clone(),
+                            payment_processing_details.paze_private_key.clone(),
+                            payment_processing_details
+                                .paze_private_key_passphrase
+                                .clone(),
+                        )
+                        .change_context(errors::ApiErrorResponse::InternalServerError)?,
```

Comment:
Include context message with `attach_printable` / `attach_printable_lazy` whenever raising internal server errors, here and elsewhere in this file.

Reply:
Done